src/luascript.cpp

```diff
@@ -4413,12 +4438,76 @@ int LuaScriptInterface::luaGameStartRaid(lua_State* L)
 	const std::string& raidName = getString(L, 1);
 
 	Raid* raid = g_game.raids.getRaidByName(raidName);
-	if (raid) {
-		raid->startRaid();
-		pushBoolean(L, true);
+	if(!raid || !raid->isLoaded()) {
+		lua_pushnumber(L, RETURNVALUE_NOSUCHRAID);
+		return 1;
+	} else if (g_game.raids.getRunning()) {
+		lua_pushnumber(L, RETURNVALUE_ANOTHERRAIDEXECUTING);
+		return 1;
+	}
+
+	RaidEvent* raidEvent = raid->getNextRaidEvent();
+	if (!raidEvent) {
+		lua_pushnumber(L, RETURNVALUE_RAIDCONTAINSNODATA);
+		return 1;
+	}
+
+	g_game.raids.setRunning(raid);
+	raid->startRaid();
+	lua_pushnumber(L, RETURNVALUE_NOERROR);
+	return 1;
+}
+
+int LuaScriptInterface::luaGameReload(lua_State* L)
+{
+	// Game.reload(interfaceName)
+	const std::string& interfaceName = asLowerCaseString(getString(L, 1));
+	
+	if (interfaceName == "action" || interfaceName == "actions") {
+		g_actions->reload();
+	} else if (interfaceName == "config" || interfaceName == "configuration") {
+		g_config.reload();
+	} else if (interfaceName == "creaturescript" || interfaceName == "creaturescripts") {
+		g_creatureEvents->reload();
+	} else if (interfaceName == "monster" || interfaceName == "monsters") {
+		g_monsters.reload();
+	} else if (interfaceName == "move" || interfaceName == "movement" || interfaceName == "movements"
+		|| interfaceName == "moveevents" || interfaceName == "moveevent") {
+		g_moveEvents->reload();
+	} else if (interfaceName == "npc" || interfaceName == "npcs") {
+		Npcs::reload();
+	} else if (interfaceName == "raid" || interfaceName == "raids") {
+		g_game.raids.reload();
+		g_game.raids.startup();
+	} else if (interfaceName == "spell" || interfaceName == "spells") {
+		g_spells->reload();
+		g_monsters.reload();
```

Comment:
This issue is not related to this PR per se, but it would be good to have a separate issue to understand better.